test/network.bats

```diff
@@ -127,9 +127,12 @@ function teardown() {
 }
 
 @test "Clean up network if pod sandbox fails" {
+	# TODO FIXME find a way for sandbox setup to fail if manage ns is true
 	cp $(which conmon) "$TESTDIR"/conmon
-	CONMON_BINARY="$TESTDIR"/conmon
-	start_crio "" "prepare_plugin_test_args_network_conf"
+	CONTAINER_MANAGE_NS_LIFECYCLE=false \
+		CONTAINER_DROP_INFRA_CTR=false \
```

Comment:
Since false is the default for these two, why set them?

I suspect I'm missing something...

Reply:
same question to a few cases below.